### DISCUSSION TITLE
Accessing smtp_server.log

### DISCUSSION THREAD

**Discussion by ndTEC** [postalserver/postal#1639] [Help with using Postal]
Hi,
In v1 i had configured fail2ban using logpath = /opt/postal/log/smtp_server.log
This file is now in the smtp container. Is it safe to bind locally the directory /opt/postal/app/log from the smtp container?
 What will happen after an upgrade if i modify the docker-compose.yml from /opt/postal/install?

**Comment by willpower232** [comment]
You could bind the directory if you wanted, have a look at #1637 for ideas about using an override file to make it a little safer

**Reply by ndTEC** [reply]
So in the end i did like this. The only thing to keep in mind is the permissions of /opt/log/smtp_server.log . On the docker host it should have rw for others.
  smtp:
    image: ghcr.io/postalserver/postal:2.0.0
    command: postal smtp-server
    network_mode: host
    cap_add:
      - NET_BIND_SERVICE
    volumes:
      - /opt/postal/config:/config
      **- /opt/log/smtp_server.log:/opt/postal/app/log/smtp_server.log**
    restart: always
